shoot-client/network-connection.sh

```diff
@@ -44,98 +46,124 @@ function configure_tcp() {
   set_value /proc/sys/net/ipv4/tcp_retries2 $tcp_retries2
 }
 
-if [[ -z "$DO_NOT_CONFIGURE_KERNEL_SETTINGS" ]]; then
+function configure_bonding() {
+  local addr
+  local targets
+
+  if [[ "$IS_SHOOT_CLIENT" == "true" ]]; then
+    # IP address is fixed on shoot side
+    addr="$bondPrefix.$((vpn_client_index+10))/24"
+    targets="${bondPrefix}.1" # using a dummy address as kube-apiserver IPs are unknown
+  else
+    # for each kube-apiserver pod acquire an IP via consensus
+    # based on pod annotations (details see go part)
+    log "acquiring ip address for bonding"
+    OUTPUT=/tmp/acquired-ip ./acquire-ip
+    addr="$(</tmp/acquired-ip)/24"
+
+    for ((i=0; i < $HA_VPN_CLIENTS; i++))
```

Comment:
See related comment concerning `HA_VPN_CLIENTS` on seed.

Reply:
The env variables `HA_VPN_CLIENTS` and `HA_VPN_SERVERS` are only used in the function `configure_bonding`.
This function is only called if the env variable `CONFIGURE_BONDING` is set to true (used only in the init container anyway).